documentation/modules/exploit/linux/misc/hp_jetdirect_path_traversal.md

```diff
@@ -0,0 +1,73 @@
+## Vulnerable Devices
+
+  Impacted printers have a firmware version below 1708D.
+
+  | Product Name                             | Model                          |
```

Comment:
Have you tried to view this with ```info -d```? I don't think this format works well :-(

Reply:
I didn't. Thanks for pointing that out as I had quickly created Markdown that I tested with GitHub.